pkg/proc/gdbserial/undo.go

```diff
@@ -27,6 +27,7 @@ import (
 type undoSession struct {
 	checkpointNextId int
 	checkpoints      map[int]proc.Checkpoint
+	volatile         bool
```

Comment:
Could you add a comment documenting the new field?

Reply:
:+1: